File: homework1/lines.py
import sys
num_lines = 0
for line in open(sys.argv[1]):
	line = line.strip()
	if line and not line.startswith("#"):
		num_lines += 1
print num_lines

#http://stackoverflow.com/questions/7896495/python-how-to-check-if-a-line-is-an-empty-line

File: homework2/lines.js
var stdio = require('stdio');
var S = require('string');
numLines = 0;

stdio.readByLines(function lineHandler(line) {
    if (!(S(line).isEmpty() || S(line).startsWith("//"))) {
    	numLines += 1;
    }
}, function (end) {
	console.log(numLines);
})
